pkg/watcher/reconciler/dynamic/dynamic.go

```diff
@@ -342,11 +356,6 @@ func (r *Reconciler) sendLog(ctx context.Context, o results.Object) error {
 					zap.Error(err),
 				)
 			}
-			logger.Debugw("Streaming log completed",
```

Comment:
This asynchronous functions never reaches this point if there is a successful streaming of logs while we do see this when there an error while streaming logs